### PR TITLE
Deleting cors domain moved to modal

### PR DESCRIPTION
**Pull request by benkeen**
This is part of the death-to-all-window.confirm()'s that Justin's
leading. The delete origin is now a modal.

Two related things:
- The cors page wasn't scrolling, hence the addition of the flex-body
class on the page element.
- the ConfirmationModal component now lets you pass in a class
for the main button to change the style.

screen shot 2016-03-31 at 2 34 28 pm

### CODE REVIEW

**Comment by kxepal** [comment]
The message is awesome! It seems I can delete google.com now (:

File: app/addons/fauxton/components.react.jsx [hunk 1]
```diff
@@ -403,7 +405,7 @@ function (app, FauxtonAPI, React, ReactDOM, ZeroClipboard, ReactBootstrap) {
             {content}
           </Modal.Body>
           <Modal.Footer>
-            <button className="btn btn-success js-btn-success" onClick={this.props.onSubmit}>
+            <button className={btnClasses} onClick={this.props.onSubmit}>
               <i className="fonticon-ok-circled"></i> {this.props.successButtonLabel}
             </button>
```
Comment (robertkowalski) [inline app/addons/fauxton/components.react.jsx]: we have a component for that:

https://github.com/apache/couchdb-fauxton/blob/master/app/addons/components/react-components.react.jsx#L1119-L1149

Comment (benkeen) [inline app/addons/fauxton/components.react.jsx]: Yeah, I guess I can refactor that component to allow passing the custom classes.

Comment (benkeen) [inline app/addons/fauxton/components.react.jsx]: Just FYI, I did try using the ConfirmButton component but ran into a circular reference problem where one file required the other. As discussed, way back when, I'm going to combine those two files + this problem will go away.

File: app/addons/components/react-components.react.jsx [hunk 2]
```diff
@@ -1147,17 +1149,18 @@ define([
     },
 
     render: function () {
+      const { onClick, buttonType, id, style, text } = this.props;
```
Comment (robertkowalski) [inline app/addons/components/react-components.react.jsx]: <3

**Comment by robertkowalski** [comment]
+1

**Comment by benkeen** [comment]
Thanks Robert. Merged as 30fdd49

**Pull request closed without merging**